docs/README.md

```diff
@@ -176,5 +175,5 @@ To create an executable for the measurement service project follow the steps pro
 
 ## Statically registering LabVIEW measurements
 
-- Refer to the Static Registration of measurements Readme for statically registering measurements.
+- Refer to the Static Registration of the MeasurementLink Readme for statically registering measurements.
```

Comment:
I don't know what this Readme is. Is this the readme that gets installed when you install MeasurementLink using NIPM? If so, we should make that clearer.

Reply:
I'm just going to take this section out. Static registration is covered in the general MeasurementLink HELP.